acstools/acscteforwardmodel.py

```diff
@@ -8,8 +8,8 @@
 `ACS Data Handbook <http://www.stsci.edu/hst/acs/documents/handbooks/currentDHB/>`_
 for more details.
 
-For guidance on running the CTE forward model, see the Jupyter notebook `ACS \
-CTE Forward Model Example <https://github.com/spacetelescope/acs-notebook/blob\
+For guidance on running the CTE forward model, see the Jupyter notebook `ACS
+CTE Forward Model Example <https://github.com/spacetelescope/acs-notebook/blob
```

Comment:
Oh, ops. I am actually not sure if Sphinx will render URL properly if it is broken up... Did you try to build the doc and see how this is rendered? You might have to make the URL text all one line (against my earlier comment). That is this all in one line:
```
`ACS CTE Forward Model Example <url here>`_
```
Sorry for any inconvenience caused!

Reply:
I did build the doc to see if it rendered correctly, and it looked fine to me. It's possible I didn't build it correctly, though, so I will make the link all one line.